vhs-frontend/src/pages/Explore.tsx

```diff
@@ -1,68 +1,49 @@
-import { useEffect, useState } from "react";
-import axios from "axios";
-import { VHS } from "../types";
-import { VhsThumbnail } from "../components/VHSThumbnail";
-import { SearchBar } from "../components/SearchBar";
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { VHS } from '../types';
+import { VhsThumbnail } from '../components/VHSThumbnail';
+import { SearchBar } from '../components/SearchBar';
 // @ts-ignore
-import placeholder from "../assets/placeholder.jpg";
+import placeholder from '../assets/placeholder.jpg';
+import { SearchFormInput } from '../components/searchBarConfig';
 
 export const Explore = () => {
   const [vhsList, setVhsList] = useState<VHS[]>([]);
-  const [filteredList, setFilteredList] = useState<VHS[]>([]);
-  const [searching, setSearching] = useState(false);
+  const [query, setQuery] = useState('');
 
   useEffect(() => {
-    getVhsList();
-  }, []);
+    getVhsList(query);
+  }, [query]);
 
-  const getVhsList = async () => {
+  const getVhsList = async (query?: string) => {
     try {
-      const response = await axios.get("/api/vhs");
+      const response = await axios.get(`/api/vhs${query}`);
```

Comment:
When you use `axios.get`, the first argument to the function is the route, but the second argument are options where you can define the parameters for query string and axios will build it for you.

```
const response = await axios.get('/api/vhs', { params: { [data.queryType]: data.queryText } });
```

Additionally, maybe renaming `queryText` to `queryValue` is a bit more semantic.